releases/20.0.0.12/images.txt

```diff
@@ -0,0 +1,12 @@
+kernel-slim      Dockerfile.ubuntu.adoptopenjdk8    open-liberty    		          20.0.0.12-kernel-slim-java8-openj9
+kernel-slim      Dockerfile.ubuntu.adoptopenjdk11   open-liberty    		          20.0.0.12-kernel-slim-java11-openj9
+kernel-slim      Dockerfile.ubi.adoptopenjdk8       openliberty/open-liberty      20.0.0.12-kernel-slim-java8-openj9-ubi
+kernel-slim      Dockerfile.ubi.adoptopenjdk11      openliberty/open-liberty      20.0.0.12-kernel-slim-java11-openj9-ubi
+kernel-slim      Dockerfile.ubi.adoptopenjdk15      openliberty/open-liberty      20.0.0.12-kernel-slim-java15-openj9-ubi
+full             Dockerfile.ubuntu.adoptopenjdk8    open-liberty     		          20.0.0.12-full-java8-openj9
+full             Dockerfile.ubuntu.adoptopenjdk11   open-liberty     		          20.0.0.12-full-java11-openj9
+full             Dockerfile.ubi.adoptopenjdk8       openliberty/open-liberty      20.0.0.12-full-java8-openj9-ubi
+full             Dockerfile.ubi.adoptopenjdk11      openliberty/open-liberty      20.0.0.12-full-java11-openj9-ubi
+full             Dockerfile.ubi.adoptopenjdk15      openliberty/open-liberty      20.0.0.12-full-java15-openj9-ubi
+beta             Dockerfile.ubuntu.adoptopenjdk8    open-liberty    		          20.0.0.12-beta-java8-openj9
```

Comment:
beta info can be removed from here when the _beta_ folder is deleted from 20.0.0.12